cartridges/int_gtm/cartridge/templates/default/common/layout/checkout.isml

```diff
@@ -0,0 +1,53 @@
+<iscontent type="text/html" charset="UTF-8" compact="true"/>
+
+<isinclude template="/components/modules" sf-toolkit="off" />
+
+<!DOCTYPE html>
+<html lang="${require('dw/util/Locale').getLocale(request.getLocale()).getLanguage()}">
+    <head>
+        <!--[if gt IE 9]><!-->
+            <isinclude sf-toolkit="off" template="/common/scripts" />
+        <!--<![endif]-->
+        <isinclude template="/common/htmlHead" />
+
+        <isinclude url="${URLUtils.url('GtmIdSet-SetHead')}"/>
```

Comment:
Don't need to edit this file to add this include. Please use hook already defined in line 23 for this:
${dw.system.HookMgr.callHook('app.template.beforeHeader', 'beforeHeader', pdict) || ''}

The hook can be used to render a template directly, no need to override this template.

Reply:
updated with the changes